tests/model_serving/model_server/private_endpoint/test_kserve_private_endpoint.py

```diff
@@ -0,0 +1,85 @@
+from typing import Self
+
+from simple_logger.logger import get_logger
+from ocp_resources.namespace import Namespace
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.pod import Pod
+from ocp_resources.deployment import Deployment
+from tests.model_serving.model_server.private_endpoint.utils import curl_from_pod
+
+
+LOGGER = get_logger(name=__name__)
+
+
+class TestKserveInternalEndpoint:
```

Comment:
Can you add a comment at the start of the class explaining the goal of these tests, so that someone not familiar with the feature can get a basic idea of what you are testing here?